alignak/http_daemon.py

```diff
@@ -243,6 +244,10 @@ def get_socks_activity(self, socks, timeout):
     # We are asking us to stop, so we close our sockets
     def stop(self):
         self.stop_requested = True
+        for t in self.threads:
+            while t.is_alive():
+                t.join(5)
```

Comment:
> I'd more safely do :

But I'd certainly add a warning if some of those client threads are still alive after this while loop. 'cause it's not normal, in the usual case, for a client thread to stay connected that long.